bm/charter_condition.go

```diff
@@ -0,0 +1,47 @@
+package bm
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// CharterCondition response description
+type CharterCondition struct {
+	Period          *Period   `json:"period,omitempty"`
```

Comment:
I know in the document it shows period. But I think we should change it. 

Everywhere else we explicitly use `DateFrom` and `DateTo`. I'll update the doc.